frontstage/views/register.py

```diff
@@ -34,6 +35,8 @@ def validate_enrolment_code(enrolment_code):
 
     if result.status_code == 200 and json.loads(result.text)['active']:
         case_id = json.loads(result.text)['caseId']
+    elif result.status_code != 200:
```

Comment:
I think that could be a good idea for the more standard errors that we could expect from other services, 400, 404. 500 etc. For the service specific one's we'll have to handle them individually